@navikt/core/tokens/darkside/tokens/colors/semantic-roles/brand-blue.tokens.ts

```diff
@@ -6,119 +6,119 @@ export const brandBlueSemanticTokenConfig = {
     },
     "brand-blue-moderate": {
       value: "{ax.brand-blue.200.value}",
       type: "color",
       group: "background.brand-blue",
-      comment: "TODO: Sjur fyller ut",
+      comment: "En medium-svak bakgrunnsfarge som brukes til dekor. Om den brukes på meningsbærende elementer må den kombineres med en godkjent border.",
     },
     "brand-blue-moderateA": {
       value: "{ax.brand-blue.200A.value}",
       type: "color",
       group: "background.brand-blue",
-      comment: "TODO: Sjur fyller ut",
+      comment: "En medium-svak bakgrunnsfarge som brukes til dekor. Om den brukes på meningsbærende elementer må den kombineres med en godkjent border. Er delvis gjennomsiktig.",
     },
     "brand-blue-moderate-hover": {
       value: "{ax.brand-blue.300.value}",
       type: "color",
       group: "background.brand-blue",
-      comment: "TODO: Sjur fyller ut",
+      comment: "En medium-svak bakgrunnsfarge som brukes til hover-state på meningsbærende elementer. Må kombineres med en godkjent border.",
     },
     "brand-blue-moderate-hoverA": {
       value: "{ax.brand-blue.300A.value}",
       type: "color",
       group: "background.brand-blue",
-      comment: "TODO: Sjur fyller ut",
+      comment: "En medium-svak bakgrunnsfarge som brukes til hover-state på meningsbærende elementer. Må kombineres med en godkjent border. Er delvis gjennomsiktig.",
     },
     "brand-blue-moderate-pressed": {
       value: "{ax.brand-blue.400.value}",
       type: "color",
       group: "background.brand-blue",
-      comment: "TODO: Sjur fyller ut",
+      comment: "En medium-svak bakgrunnsfarge som brukes til active/selected-state på meningsbærende elementer. Må kombineres med en godkjent border.",
     },
     "brand-blue-moderate-pressedA": {
       value: "{ax.brand-blue.400A.value}",
       type: "color",
       group: "background.brand-blue",
-      comment: "TODO: Sjur fyller ut",
+      comment: "En medium-svak bakgrunnsfarge som brukes til active/selected-state på meningsbærende elementer. Må kombineres med en godkjent border. Er delvis gjennomsiktig.",
     },
     "brand-blue-strong": {
       value: "{ax.brand-blue.600.value}",
       type: "color",
       group: "background.brand-blue",
-      comment: "TODO: Sjur fyller ut",
+      comment: "En sterk bakgrunnsfarge som brukes på meningsbærende elementer.",
     },
     "brand-blue-strong-hover": {
       value: "{ax.brand-blue.700.value}",
       type: "color",
       group: "background.brand-blue",
-      comment: "TODO: Sjur fyller ut",
+      comment: "En sterk bakgrunnsfarge som brukes til hover-state på meningsbærende elementer.",
     },
     "brand-blue-strong-pressed": {
       value: "{ax.brand-blue.800.value}",
       type: "color",
       group: "background.brand-blue",
-      comment: "TODO: Sjur fyller ut",
+      comment: "En sterk bakgrunnsfarge som brukes til active/selected-state på meningsbærende elementer.",
```

Comment:
@sjur-gr Blir `meningsbærende` det rette ordet her? Tenker det meste kan regnes som meningsbærende på et vis 🤔

Reply:
Usikker. Brukte samme ord som i denne artikkelen https://aksel.nav.no/god-praksis/artikler/141-bruk-av-farge

Reply:
Mulig jeg kan linke til artikler/sider her også for å koble beskrivelsen med en bedre forklaring. Kan gå over etter hvert og legge til.

Reply:
👍 Ja skal ikke være noe problem å legge til markdown-lenker her 
```
[lenke](https://example.com)
```